### ISSUE TITLE
Verification fails: (TypeScript) Option needs to be "aud" instead "audience" 

### ISSUE THREAD

**Issue by bke-daniel** [auth0/node-jwks-rsa#85]
Hello everyone,

I'm experiencing the issue that I need to pass "aud" instead "audience" as option for the client in order to get the token decripted.

We're using the latest available versions.

Given TS interface:
```typescript
export function verify(
    token: string,
    secretOrPublicKey: string | Buffer | GetPublicKeyOrSecret,
    options?: VerifyOptions, // <---
    callback?: VerifyCallback,
): void;

export interface VerifyOptions {
    algorithms?: string[];
    audience?: string | RegExp | Array<string | RegExp>; // <---
    clockTimestamp?: number;
    clockTolerance?: number;
    issuer?: string | string[];
    ignoreExpiration?: boolean;
    ignoreNotBefore?: boolean;
    jwtid?: string;
    subject?: string;
    /**
     * @deprecated
     * Max age of token
     */
    maxAge?: string;
}
```
Verify call:
```TypeScript
const options /*: VerifyOptions */ = {
  aud: config.AUTH0_AUDIENCE, // <---
  issuer: `https://${config.AUTH0_DOMAIN}/`,
  algorithms: ["RS256"],
};

const identity = new Promise((resolve, reject) => {
  jwt.verify(token, getKey, options, (err, decoded) => {
    if (err) {
      return reject(err);
    }
    resolve(decoded);
  });
});
```

**TL;DR; Problem:**
* Interface defines "audience", failes
* No interface, but option passed as "aud" succeeds

Thank you guys,
Daniel
